src/components/Person/Person.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  let partnerString = `wife's`;
+  let marriedString = 'I am not married';
+
+  if (person.sex === 'f') {
+    partnerString = `husband's`;
+  }
+
+  if (person.isMarried) {
+    marriedString
+      = `My ${partnerString} name is ${person.partnerName}`;
+  }
```

Comment:
You can use ternary operator to set some value to the variables depending on some check